ScoutSuite/frontend/src/partials/gcp/computeengine.projects.id.zones.id.instances.id/index.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import { Link } from 'react-router-dom';
+
+import { Partial, PartialValue } from '../../../components/Partial';
+import {
+  partialDataShape,
+  formatDate,
+  convertBoolToEnable,
+} from '../../../utils/Partials';
+import { TabsMenu, TabPane } from '../../../components/Tabs';
+import GetResourceName from '../../../components/GetResourceName';
+import { valueOrNone } from '../../../utils/Partials/index';
+
+const renderTags = tags => {
```

Comment:
i made a `renderTags` utils function on my side.

Reply:
I saw that 🙂 This one seems weirdly complicated, I don't remember why...